bin/tx-prover/src/commands/proxy.rs

```diff
@@ -18,6 +18,7 @@ impl StartProxy {
     ///
     /// This method will first read the config file to get the list of workers to start. It will
     /// then start a proxy with each worker as a backend.
+    #[tracing::instrument(target = "miden-tx-prover", name = "proxy:execute")]
```

Comment:
We should now be able to use constants here (e.g., `MIDEN_TX_PROVER`) instead of string literals.